src/lib/aggregations/get-avg.ts

```diff
@@ -1,15 +1,17 @@
-import { Document, Field } from 'lib/common'
+import { Document } from 'lib/common'
+
+type NumericField<TDocument extends Document> = {
```

Comment:
this could be moved to common

Reply:
https://github.com/codemask-labs/nestjs-elasticsearch/blob/main/src/lib/common/field.ts